README.md

```diff
@@ -4,43 +4,40 @@
 - Node >= 6
 - NPM
 
-
 ### Install
 
-- Clone repository
-    
-    `git clone https://github.com/testica/a3hrgo-assistant`
-
-- Inside project install dependencies
-
-    `npm install`
+    `npm install -g a3hrgo`
```

Comment:
Did you publish to npm ?

Reply:
I noticed that first we need to merge this and then publish

Reply:
I have not published it, I hope you do publish it